servers/team_allocation/db/migration/0006_tutor.up.sql

```diff
@@ -0,0 +1,20 @@
+BEGIN;
+
+CREATE TABLE
+    tutor
```

Comment:
Open for discussion: Would it be a good idea to also add the University ID here? Maybe we need that down the line? - This can definitely be a follow up i guess